samples/bluetooth/channel_sounding_ras_initiator/README.rst

```diff
@@ -0,0 +1,98 @@
+
+A basic distance estimation algorithm is included in the sample.
+The mathematical representations described in [#phase_and_amplitude]_ and [#rtt_packets]_ are used as the basis for this algorithm.
+
+User interface
+**************
+
+The sample does not require user input and will scan for a device advertising with the GATT Ranging Service UUID.
+The first LED on the development kit will be lit when a connection has been established.
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/bluetooth/channel_sounding_ras_initiator`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After programming the sample to your development kit, you can test it by connecting to another development kit with a Channel Sounding Reflector role with Ranging Responder.
+
+1. |connect_terminal_specific|
+#. Reset the kit.
+#. Program the other kit with the Channel Sounding Reflector with Ranging Responder sample.
+#. Wait until the scanner detects the Peripheral.
+   In the terminal window, check for information similar to the following::
+
+      Filters matched. Address: XX:XX:XX:XX:XX:XX (random) connectable: 1
+      Connecting
+      Connected to XX:XX:XX:XX:XX:XX (random) (err 0x00)
+      Security changed: XX:XX:XX:XX:XX:XX (random) level 2
+      MTU exchange success (498)
+      The discovery procedure succeeded
+      CS capability exchange completed.
+      CS config creation complete. ID: 0
+      CS security enabled.
+      CS procedures enabled.
+      Subevent result callback 0
+      Ranging data ready 0
+      Ranging data get completed for ranging counter 0
+      Estimated distance to reflector:
+      - Round-Trip Timing method: X.XXXXX meters (derived from X samples)
+      - Phase-Based Ranging method: X.XXXXX meters (derived from X samples)
+
+Dependencies
+************
+
+This sample uses the following |NCS| libraries:
+
+* :ref:`dk_buttons_and_leds_readme`
+* :file:`include/bluetooth/gatt_dm.h`
+* :file:`include/bluetooth/services/ras.h`
+
+This sample uses the following Zephyr libraries:
+
+* :file:`include/sys/printk.h`
+* :file:`include/zephyr/types.h`
+* :ref:`zephyr:kernel_api`:
+
+  * :file:`include/kernel.h`
+
+* :ref:`zephyr:bluetooth_api`:
+
+* :file:`include/bluetooth/bluetooth.h`
+* :file:`include/bluetooth/conn.h`
+* :file:`include/bluetooth/cs.h`
+
+References
+**********
+
+.. [#phase_and_amplitude] `Bluetooth Core Specification v. 6.0: Vol. 1, Part A, 9.2 <https://www.bluetooth.com/wp-content/uploads/Files/Specification/HTML/Core-60/out/en/architecture,-change-history,-and-conventions/architecture.html#UUID-a8d03618-5fcf-3043-2198-559653272b1b>`_
+.. [#rtt_packets] `Bluetooth Core Specification v. 6.0: Vol. 1, Part A, 9.3 <https://www.bluetooth.com/wp-content/uploads/Files/Specification/HTML/Core-60/out/en/architecture,-change-history,-and-conventions/architecture.html#UUID-9d4969af-baa6-b7e4-03ca-70b340877adf>`_
```

Comment:
Please place these to the links.txt (docs/nrf). See https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/dev_model_and_contributions/documentation/styleguide.html#linking